server/db/models/message.js

```diff
@@ -10,6 +10,10 @@ const Message = db.define("message", {
     type: Sequelize.INTEGER,
     allowNull: false,
   },
+  read: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: true,
```

Comment:
Because we have a default value, I think it is a good idea to add `allowNull: false` here to be safe.